src/index.ts

```diff
@@ -102,6 +102,13 @@ export default class OtaClient {
         return this.locale;
     }
 
+    /**
+     * List manifest timestamp of distribution
```

Comment:
Looks like a typo here. We are not retrieving any list here. Maybe change it to `Get the timestamp of the manifest` .